hasura/migrations/default/1717008407293_alter_table_public_colinks_gives_add_column_tx_hash/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."colinks_gives" add column "tx_hash" text
```

Comment:
is not it better to add all changes of the same table of the same feature to the same migration file?

Reply:
it is better - but im lazy now